crates/optimism/txpool/src/supervisor/metrics.rs

```diff
@@ -9,6 +13,31 @@ use std::time::Duration;
 pub struct SupervisorMetrics {
     /// How long it takes to query the supervisor in the Optimism transaction pool
     pub(crate) supervisor_query_latency: Histogram,
+
+    /// Counter for the number of times the chain database was uninitialized
+    pub(crate) uninitialized_chain_database_count: Counter,
+    /// Counter for the number of times data was skipped
+    pub(crate) skipped_data_count: Counter,
```

Comment:
I think we can create a single metric 'supervisor_api_errors' and have the error_type as tags to the metric. Let me know what you think.

Reply:
what are the benefits? link to docs for the type you are referring to?